pipeline.ipynb

```diff
@@ -520,9 +538,9 @@
    "source": [
     "%%opts Curve [width=500, tools=['hover']]\n",
     "hv.output(size=output_size)\n",
-    "if interactive and in_memory:\n",
-    "    hv.NdOverlay(dict(width=hv.Curve(shifts.sel(variable='width')),\n",
-    "                      height=hv.Curve(shifts.sel(variable='height'))))"
+    "if interactive: #and in_memory:\n",
```

Comment:
`in_memory` is gone in `dev` and you can safely take it out forever

Reply:
This has been removed in commit 4b9997b